src/OnePlusBot/Modules/Help.cs

```diff
@@ -20,6 +20,7 @@ public HelpModule(IServiceProvider services, CommandService commands)
         }
 
         [Command("help")]
+        [Alias("h")]
```

Comment:
`h` is an uncommon abbreviations for help; I would use ? but it might create some confusion as it's often used as a bot prefix.